tools/postman/Makefile

```diff
@@ -29,6 +29,10 @@ build: fetch_openapi convert_to_collection
 .PHONY: build_and_upload
 build_and_upload: build upload_collection
 
+.PHONY: compare_and_send_forks
+compare_and_send_forks: 
+	./scripts/send-fork-data.sh
+
```

Comment:
Could you split this to have  `make compare` and `make send-data` and then  `make compare_and_send_forks` will call both make commands. 
the reason is that you may want to compare without sending the data